launcher/src/main/java/org/apache/spark/launcher/SparkLauncher.java

```diff
@@ -211,10 +211,10 @@ public SparkLauncher addSparkArg(String arg) {
    * Adds an argument with a value to the Spark invocation. If the argument name corresponds to
    * a known argument, the code validates that the argument actually expects a value, and throws
    * an exception otherwise.
-   * <p/>
+   * <p>
```

Comment:
Hm I though that's because it now required valid HTML, like  `<p>...</p>`. Well, if it fixes an error, obviously that's better than an error. There seem to be ~17 occurrences of this in the code base though -- worth fixing it in one go?